module/VuFind/tests/unit-tests/src/VuFindTest/View/Helper/Root/RecordDataFormatterTest.php

```diff
@@ -521,9 +521,9 @@ public function testFormatting(string $function): void
             'Language' => 'ItalianLatin',
             'Published' => 'Centro di Studi Vichiani, 1992',
             'Edition' => 'Fictional edition.',
-            'Series' => 'Vico, Giambattista, 1668-1744. Works. 1982 ;',
             'Multi Count' => 1,
             'Multi Data' => 'Book',
+            'Series' => 'Vico, Giambattista, 1668-1744. Works. 1982 ;',
```

Comment:
Is this change related to the rest of the PR? It certainly doesn't hurt to put the expectations in alphabetical order, but I don't see a reason why your other code changes would require this test to be updated. Did you start to work on adding a test case for the new functionality but decide not to proceed with it?

Reply:
thanks @demiankatz yes this was due to me starting to work on adding a test case for the new functionality and then I decided not to proceed with it. I've removed this change in my latest follow up